.gitban/cards/archive/sprints/20260609-231157-sprint-superseded/backlog-P2-chore-case-insensitive-external-integration-id-matching-follow-up-from-ca0ios-CAMERON.md

```
@@ -0,0 +1,78 @@
+## Work Log
+
+|            Step             | Status/Details        |                  Universal Check                  |
+| :-------------------------: | :-------------------- | :-----------------------------------------------: |
+| **1. Review Current State** | Not started — backlog | - [ ] Current state is understood and documented. |
+|     **2. Plan Changes**     | Not started — backlog |         - [ ] Change plan is documented.          |
+|     **3. Make Changes**     | Not started — backlog |          - [ ] Changes are implemented.           |
+|     **4. Test/Verify**      | Not started — backlog |        - [ ] Changes are tested/verified.         |
+| **5. Update Documentation** | Not started — backlog |  - [ ] Documentation is updated [if applicable].  |
+|     **6. Review/Merge**     | Not started — backlog |      - [ ] Changes are reviewed and merged.       |
+
+#### Work Notes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)